doc/man1/ca.pod

```diff
@@ -262,6 +263,12 @@ configuration file, must be valid UTF8 strings.
 If reading serial from the text file as specified in the configuration
 fails, specifying this option creates a new random serial to be used as next
 serial number.
+To get random serial numbers, use the B<-rand_serial> flag instead; this
+should only be used for simple error-recovery.
+
+=item B<-rand_serial>
+
+Generate a large random number to use as the serial number.
```

Comment:
You should mention that this overrides any serial file option (`-create_serial` and configuration key `serial`)